docker/windows/runner/Dockerfile

```diff
@@ -15,14 +15,21 @@ FROM cobalt-build-win32-base
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';$ProgressPreference='silentlyContinue';"]
 
-ARG RUNNER_VERSION
+ARG RUNNER_VERSION=2.298.2
```

Comment:
Doesn't this just set the default value? And wouldn't be override that somewhere else as it didn't have a default value before?

Reply:
Good catch! The RUNNER_VERSION is actually set in the docker-compose file.

Reply:
I am going to close this ticket for now and bump windows runner version when we upgrade ARC in our cluster.  Having said that, it looks like it doesn't matter what version is installed, the runner will detect the version to be outdated and will automatically upgrade to latest and greatest.  I still think we should regularly (whenever we upgrade ARC?) bump runner version.